sys-fs/fragview/files/fragview-0.1.1-fix-linking.patch

```diff
@@ -0,0 +1,40 @@
+From 11ab3027fda015798b92fd7e44cb5e3289f41542 Mon Sep 17 00:00:00 2001
```

Comment:
Please include link to upstream bug tracker so we can monitor its status.

Reply:
done